roaringbitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -70,7 +73,7 @@ public class Roaring64NavigableMap implements Externalizable, LongBitmapDataProv
 
   // As of RoaringBitmap 0.X, we stick to the legacy format for retrocompatibility
   // RoaringBitmap 1.X may switch to the portable format by default
-  public static int SERIALIZATION_MODE = SERIALIZATION_MODE_LEGACY;
```

Comment:
I disagree with dropping the `static ` flag, as it means you prevent someone to set `SERIALIZATION_MODE ` once and for all (in a JVM lifetime). You may add an additional `non-static` field, to enable having specific bitmaps with a different flag than the default one.

Reply:
You should then add documentation about this impact: changing the static flag would impact only FUTURE bitmaps, not the ones already created. Hence making it very important to cal lthe flag very early. This is all but a trivial change.

Reply:
then we will get something like this?
public static final int SERIALIZATION_MODE_LEGACY = 0;
public static final int SERIALIZATION_MODE_PORTABLE = 1;
public static int SERIALIZATION_MODE = SERIALIZATION_MODE_LEGACY;
private int serializationMode = SERIALIZATION_MODE; // or just leave the initial value as 'null'